lib/Struct.js

```diff
@@ -49,18 +49,18 @@ function Struct(name, defs) {
         }, {}));
       }
 
-      static fromBuffer(buf, i, returnLength) {
+      static fromBuffer(buf, i, returnLength, options) {
```

Comment:
I'm not sure about the changes to Struct, for consistency its nice but I don't think we need it to "fix" the stringEncoding of zigbee attributes